index.php

```diff
@@ -29,7 +44,14 @@ function drawCats(array $cats) :string {
             <div class="selector">
               <select name="breed" id="select-breed">
                 <option value="0">Please select:</option>
-                <?php ?>
+                <?php
+
+                for($i = 0; $i < count($breeds); $i++) {
```

Comment:
Rather than inline for loop, look at how $catshtml is echoed below.

I have been advised previously to keep all the logic at the top of the file and only echo strings in the HTML part.